<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OnlineWgt</class>
 <widget class="QWidget" name="OnlineWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gBox">
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="2" column="2">
     <widget class="QWidget" name="ofPgPnl" native="true"/>
    </item>
    <item row="2" column="4">
     <widget class="QWidget" name="onPgPnl" native="true"/>
    </item>
    <item row="1" column="2">
     <widget class="QWidget" name="ofPointsPnl" native="true"/>
    </item>
    <item row="2" column="6">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="5">
     <widget class="QWidget" name="onLevelFigurePnl" native="true"/>
    </item>
    <item row="3" column="3">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <widget class="QWidget" name="ofLevelFigurePnl" native="true"/>
    </item>
    <item row="1" column="4">
     <widget class="QWidget" name="onPointsPnl" native="true"/>
    </item>
    <item row="0" column="3">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="topYellowLbl">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="yellowLbl">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
